python/sdk/test/conftest.py

```diff
@@ -136,7 +136,7 @@ def requests():
     retry_strategy = Retry(
         total=5,
         status_forcelist=[429, 500, 502, 503, 504, 404],
-        method_whitelist=["POST"],
+        allowed_methods=["POST"],
```

Comment:
The change in this argument is done because the keyword argument `method_whitelist` has been [deprecated](https://github.com/GeneralMills/pytrends/issues/591) as of `urllib3>=2`. `allowed_methods` was introduced as a [replacement](https://github.com/urllib3/urllib3/issues/2092) from `urllib3>=1.26` onwards (the arguments for `method_whitelist` get passed to `allowed_methods` under the hood anyway).

Reply:
Thanks for fixing this!